content/cloud-hosting/azure/azure-service-bus.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Azure Service Bus
+showMetadata: true
+editable: true
+showToc: true
+tocDepth: 3
+order: 1
+---
+
+# การเขียน C# เพื่อจัดการ Message Queue ผ่าน Azure Service Bus และ RabbitMQ ด้วย Code ชุดเดียวกัน
+
+![Azure Service Bus and RabbitMQ](https://miro.medium.com/max/1400/1*mkmcrXnun-271tM9pXlX4w.png)
+
+เมื่อสัปดาห์ที่แล้ว ได้มีโอกาศรับงานต่อจากเพื่อนมา เพื่อจะนำระบบ Message Queue มาใช้งานจริงใน project ที่กำลังดำเนินการอยู่ ซึ่งเพื่อนก็ทำการบ้านมาเป็นอย่างดี มีตัวอย่างการใช้งาน Azure Service Bus มาให้
+
+แต่อย่างไรก็ตาม สุดท้ายผมติดปัญหาการนำมาใช้จริง เนื่องจากการ implement ด้วย Azure Service Bus สามารถทำงานได้บน Cloud แต่ถ้าเราจะทดสอบระบบ ระหว่างที่ debug ในเครื่องตัวเอง, การจะเปิด Queue บน Cloud สำหรับทุกคน นั่นค่อนข้างลำบากและมีค่าใช้จ่าย จึงจำเป็นต้องหาหนทาง ในการทำให้ทำงานใน local ได้, และ deploy ไปใช้งานร่วมกับ Cloud Service ได้ โดยใช้ code ชุดเดียวกัน โดยในที่สุด ผมก็ทำได้ โดยเก็บ source code ไว้ที่นี่ นะคับ
+
+ ```
+https://github.com/nakornttss/azureservicebusrabmitmqamqp
```

Comment:
Please use Markdown link in stead of code block. You can also use a link URL as a link text if you want.
e.g. [https://github.com/nakornttss/azureservicebusrabmitmqamqp](https://github.com/nakornttss/azureservicebusrabmitmqamqp)

However, if you really need a link text to be the same as link path. You just only put a link and our framework is smart enough to create a link text with URL path.